usmqe/web/tendrl/auxiliary/models.py

```diff
@@ -3,16 +3,17 @@
 """
 
 from webstr.core import WebstrModel, By, PageElement, RootPageElement,\
-    DynamicWebstrModel, DynamicPageElement
+    DynamicPageElement
 from webstr.common.form import models as form
 #
 # from usmqe.web.utils import StatusIcon
 
 
-class BaseListMenuModel(DynamicWebstrModel):
+class BaseListMenuModel(WebstrModel):
     """
     base auxiliary model for list menu (filter and order fields)
     """
+    _context = ""
```

Comment:
please remove this as the solution of the problem is not by using this attribute any more